src/Rings/mpoly.jl

```diff
@@ -455,10 +455,18 @@ mutable struct MPolyIdeal{S} <: Ideal{S}
   end
 end
 
-@enable_all_show_via_expressify MPolyIdeal
-
-function AbstractAlgebra.expressify(a::MPolyIdeal; context = nothing)
-  return Expr(:call, :ideal, [expressify(g, context = context) for g in collect(a.gens)]...)
```

Comment:
please don't remove the expressify definition. If you don't want it for printing, just delete the line `@enable_all_show_via_expressify MPolyIdeal`